include/lsst/afw/image/PhotoCalib.h

```diff
@@ -167,84 +175,81 @@ class PhotoCalib : public table::io::PersistableFacade<PhotoCalib>, public table
     }
 
     /**
-     * Convert instFlux in ADU to maggies at a point in the BoundedField.
+     * Convert instFlux in ADU to nJy at a point in the BoundedField.
      *
      * If passed point, use the exact calculation at that point, otherwise, use the mean scaling factor.
      *
      * @param[in]  instFlux The source instFlux in ADU.
      * @param[in]  point    The point that instFlux is measured at.
      *
-     * @returns    The flux in maggies.
+     * @returns    The flux in nJy.
      */
-    double instFluxToMaggies(double instFlux, lsst::geom::Point<double, 2> const &point) const;
+    double instFluxToNanojansky(double instFlux, lsst::geom::Point<double, 2> const &point) const;
 
-    /// @overload instFluxToMaggies(double, lsst::geom::Point<double, 2> const &) const;
-    double instFluxToMaggies(double instFlux) const;
+    /// @overload instFluxToNanojansky(double, lsst::geom::Point<double, 2> const &) const;
+    double instFluxToNanojansky(double instFlux) const;
```

Comment:
I know this was discussed on the RFC, but now seeing "nJy" as the units and "instFluxToNanojansky()" as the camelCase method, I don't like it.  Is "ToNanoJansky()" really not an option?

Reply:
If you're asking for `instFluxToNanoJansky` here, that's truly not a valid name: the unit name is `jansky` and the abbreviation is `Jy`: https://en.wikipedia.org/wiki/Jansky

If you meant `toNanojansky`, with no prefix, I don't like that because it doesn't tell you what the input to the method is (a significant flaw in the existing `Calib` object).

Reply:
Welp, I guess it just will have to look odd and asymmetric.